gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -66,14 +66,14 @@
   private final Storage storage;
 
   // used only when we create a new instance of CloudStorageFileSystemProvider.
-  private static StorageOptions storageOptions;
+  private static StorageOptions defaultStorageOptions;
```

Comment:
This is dead code walking, as jart's PR will completely change the defaults mechanism. I'll leave this line as-is to reduce upcoming merge pain.